src/main/java/uk/gov/hmcts/dts/fact/repositories/CourtLocalAuthorityRepository.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.dts.fact.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import uk.gov.hmcts.dts.fact.entity.CourtLocalAuthority;
+
+import java.util.List;
+
+
+public interface CourtLocalAuthorityRepository extends JpaRepository<CourtLocalAuthority, Integer> {
```

Comment:
May be this one should be renamed to CourtLocalAuthorityAreaOfLawRepository as the database table is named like that.

Reply:
removed accordingly